lib/utils.bash

```diff
@@ -365,8 +365,9 @@ parse_asdf_version_file() {
 
   if [ -f "$file_path" ]; then
     local version
-    # this is actually only about 5% faster than the original, despite losing a function call and two pipes
+    # this is causing multiple failures in tests/utils.bash, despite seeming to work identically
```

Comment:
Need to figure out why this works fine for actual tests (including paths with spaces in them), but fails in bats.

Reply:
Done. A combination of only printing `$NF` with awk, and then having leading whitespace.